CHANGELOG

```diff
@@ -16,6 +17,7 @@ v3.17 (Month 2020)
     - Sidebar items not showing active state
     - Stop bin/setup from creating folders outside dradis-ce/
     - bin/setup will not error if the attachments directory already exists
+    - Fix textile preview not showing on issues with very long text
```

Comment:
Can you please address the :alpha

Reply:
CHANGELOG in alpha? Now I've seen it all 🙈

Will do.